packages/ramldoc-generator/README.md

```diff
@@ -1,5 +1,7 @@
 # commercetools Docs Kit "RAMLdoc" Generator
 
+> **DEPRECATED** in favor of the new **@commercetools-docs/rmf-codegen** package which provides the complete functionality of rmf-codegen without a restrictive wrapper CLI.
+
```

Comment:
Suggestion: we can remove the rest of the README and document how to migrate to the new package.